src/api/get-installations.js

```diff
@@ -8,6 +8,24 @@ import ALIASES from '../aliases';
 // Installation info cache
 var installationsCache = null;
 
+// Original code page value
+var origCodePage = null;
```

Comment:
Never use global vars in async envirionment. Imagine the following scenario:
there are two `getInstallations` running at the same time, the first one sets `origCodePage` to the initial. Then second one tries to save original code page, but it's already modified by the first one. Think how we can avoid this.